README.md

```diff
@@ -38,6 +31,7 @@ foo@bar:~$ git clone github.com/CMakePP/CMakeDoc.git
 foo@bar:~$ cd CMakeDoc/
 foo@bar:~/CMakeDoc$ python3 -m venv virt-env #Create our virtual environment
 foo@bar:~/CMakeDoc$ source virt-env/bin/activate #Activate virtual environment
+(virt-env) foo@bar:~/CMakeDoc$ pip3 install wheel #Necessary dependency to install manually with pip
```

Comment:
I admittedly do not have any experience actually making a pip package, but my experience installing them has been that you usually don't have to preinstall dependencies because pip will go get them for you. Is there a way to do this step automatically for the user?

Reply:
Normally pip does autoinstall dependencies, and in fact already handles that for the Antlr4 runtime library that cmakedoc depends on. Unfortunately, the wheel package in particular does not work that way because it is a prerequisite to installing the package itself. In Python, wheels are the new binary distribution format, replacing the older eggs and analogous to Debian's .deb files. The error you encounter when installing without wheel is from Pip attempting to build such a binary distribution.

This problem does not occur when using the system python installation as wheel is shipped in the dist-packages, but a virtual environment (specifically the venv builtin module) does not include it as dist-packages are implicitly removed from the search path. A possible workaround would be putting all of these commands in a single bash script that the user can just run, or switching to virtualenv (a 3rd-party program that does install wheel by defaul, otherwise it is virtually identical to venv).